testsuite/meta/converter/independent_component_analysis_fast.dat

```diff
@@ -1,8 +1,10 @@
740666652}{0.11302258065583}{-0.060726720282793}{0.09691387071415984}{0.07145111735142441}{0.1271644837437242}{0.0685308004090391}{0.08065407323611661}{0.0692658350313769}{0.09940596283007845}{0.06940617877168265}{0.08423083331894621}{0.109000375394226}{0.05665666759569225}{0.06474250192816408}{0.1082321497125129}{0.07630461182521685}{0.08510467896526472}{0.07216560146016396}{0.1362462859816253}{0.04264178432970916}{0.09526358679786788}{0.05654824965767088}{0.1049359443547267}{0.04952480563813468}{0.08162428055889531}{0.06206097297359477}{0.105149533891069}{0.05403898522664009}{0.06644563237917882}{0.05223420560923143}{0.08655769832541385}{0.07724783239634381}{0.02233792046423348}{0.08437163768212005}{0.06726362127466104}{0.05453378434126852}{0.05602499434894666}{0.06205848881442502}{0.05927014180125331}{0.06505734933025339}{0.07162327426734208}{0.04736545886967568}{0.03873249337444434}{0.07933463613029529}{0.01582072669472771}{0.07596697980386075}{0.04735440984733832}{0.08870172879207466}{5.483103343485729e-05}{0.07156719934703543}{0.009690926498690397}{0.05454096814239545}{0.01996776947505356}{0.07977003916530359}{0.006370609790566667}{-0.07057177162475362}{-0.00878348954217234}{-0.06549781308747871}{-0.01614004086454267}{-0.06809263673801402}{-0.01136462111674889}{-0.05823686452413586}{-0.03884877945252359}{-0.05754425829493459}{-0.01709857373983885}{-0.06526288211501247}{-0.04191088716875362}{-0.1012471747957747}{-0.0388248955383077}{-0.08298505607025269}{-0.07597232064444906}{-0.05393478189337874}{-0.08241306031704976}{-0.0540960040965023}{-0.01782651435994625}{-0.0780148690559993}{-0.04097524766386208}{-0.07618656532706897}{-0.06580463392649595}{-0.09330153105314701}{-0.1065080299737599}{-0.06456875973462439}{-0.0754261864681766}{-0.05357846525712924}{-0.08229704921168279}{-0.06662680925696049}{-0.08556938199002288}{-0.07273531106378542}{-0.1057313835469801}{-0.07569644673402812}{-0.08130725085107719}{-0.05055848641072917}{-0.1233818117272909}{-0.06153276334268105}{-0.08254020860302877}{-0.06328973227648391}{-0.05765555435635934}{-0.06353765290135183}{-0.08223043413372437}{-0.07911931699210259}{-0.1039452298627243}{-0.07087360119507979}{-0.1053760617435425}{-0.07555702883169996}{-0.08307905873206127}{-0.07275872841383177}{-0.08449564523462133}{0.07363632498026496}{-0.07894239345732289}{0.08388416683311704}{-0.08462091292286902}{0.08193298150851998}{-0.08956217098046811}{0.07154809825765987}{-0.06704298868473958}{0.06155622825362517}{-0.07418934217404434}{0.09009060386193998}{-0.04829114550327392}{0.05646230212639323}{-0.0117919235415138}{0.06381466423670457}{0.005552033244161023}{0.08361426041962597}{-0.03315440734484235}{0.09186437686136162}{-0.004737539968636351}{0.08970137279827656}{-0.04798901202637026}{0.06639724826608545}{-0.01873521784156308}{0.07217704364707425}{0.006481368372198953}{0.09186607620774861}{0.01397221509420196}{0.08011276575513913}{0.0005471670133378459}{0.05353299536044679}{-0.004080108797745487}{0.05746999708927086})
```

Comment:
where did this line go? You didnt change anything in the cookbook, so I dont understand why this would be deleted

Reply:
fixed. I should use non-in-place mode of the transformer.